UI/menu/expanding.css

```diff
@@ -8,3 +8,28 @@ li.menu_closed > ul {
 li.menu_open > ul {
 	display: block;
 }
+
+#version_info {
+    text-align: center;
+}
+
+.header_table {
+    margin-left: auto;
+    margin-right: auto;
+    border=none; border-height=0; padding-top=0; padding-bottom=0; margin-top=0; margin-bottom=0;
```

Comment:
This is invalid CSS syntax, as far as I know. CSS uses a colon, not an equation sign.